kolibri/plugins/learn/assets/src/views/points-popup/index.vue

```diff
@@ -48,11 +55,13 @@
       document: 'Document',
       html5: 'HTML5 app',
       item: 'Item',
-      close: 'Close',
+      close: 'Practice again',
```

Comment:
String freeze, and this seems inapplicable to non-exercise types that we are showing this popup for, like videos and audio.